packages/api/package.json

```diff
@@ -27,7 +27,8 @@
     "jsonwebtoken": "^9.0.0",
     "superjson": "1.9.1",
     "trpc-openapi": "^1.2.0",
-    "viem": "^2.17.4"
+    "viem": "^2.17.4",
+    "@floating-ui/react": "*"
```

Comment:
Let’s use `pnpm add` to install the package so that it adds a locked version, allowing only non-breaking updates.

Using `*` is not recommended, as it installs any new version, including those with potential breaking changes.

Reply:
Done [2ed8ac7](https://github.com/Blobscan/blobscan/pull/507/commits/2ed8ac787db049b63c5a841a25987f868df0975d)